pt-BR/mail/chrome/messenger/addressbook/abResultsPaneOverlay.dtd

```diff
@@ -68,13 +68,13 @@
 <!ENTITY sortDescending.accesskey              "r">
 
 <!-- context menu -->
-<!ENTITY composeEmail.label                    "Compose Email To">
+<!ENTITY composeEmail.label                    "Criar email para">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY composeEmail.label                    "Criar mensagem de correio eletrônico para">